src/compile/compile.ts

```diff
@@ -18,6 +18,7 @@ import {
 import {Dict, keys} from '../util';
 import {buildModel} from './buildmodel';
 import {assembleRootData} from './data/assemble';
+//import {drawDataflow} from './data/debug';
```

Comment:
These unrelated changes should not be in the pull request.

Reply:
Resolved.